### PR TITLE
fix: should print logger when agent start error

### PR DESCRIPTION
**Pull request by nightink**
##### Checklist

- [x] `npm test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines

##### Description of change

Close https://github.com/eggjs/egg/issues/612

### CODE REVIEW

File: lib/agent_worker.js [hunk 1]
```diff
@@ -21,8 +21,8 @@ agent.ready(() => {
 
 // exit if agent start error
 agent.once('error', startErrorHanddler);
-function startErrorHanddler() {
-  consoleLogger.error('[agent_worker] Agent Worker start error, exiting now!');
+function startErrorHanddler(error) {
+  consoleLogger.error('[agent_worker] Agent Worker start error, exiting now!', error);
```
Comment (fengmk2) [inline lib/agent_worker.js]:
```js
consoleLogger.error(err);
consoleLogger.error('[agent_worker] Agent Worker start error, exiting now!');
```

分开打印

Comment (nightink) [inline lib/agent_worker.js]: done

File: lib/agent_worker.js [hunk 2]
```diff
@@ -21,7 +21,8 @@ agent.ready(() => {
 
 // exit if agent start error
 agent.once('error', startErrorHanddler);
-function startErrorHanddler() {
+function startErrorHanddler(err) {
+  consoleLogger.error(err);
```
Comment (popomore) [inline lib/agent_worker.js]: 要打 stack 么

Comment (nightink) [inline lib/agent_worker.js]: 嗯 这样能一眼看到错误位置。

不打的考虑是什么

Comment (nightink) [inline lib/agent_worker.js]: image

Comment (fengmk2) [inline lib/agent_worker.js]: consoleLogger.error(err); 会处理

**Comment by fengmk2** [comment]
1.6.2

**Pull request merged**